Assets/MixedRealityToolkit.SDK/Experimental/Features/Utilities/TapToPlace/HideTapToPlaceLabel.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using Microsoft.MixedReality.Toolkit.Experimental;
+using Microsoft.MixedReality.Toolkit.Experimental.Utilities;
+using Microsoft.MixedReality.Toolkit.Utilities.Solvers;
+using Microsoft.MixedReality.Toolkit.Utilities;
+using UnityEngine;
+
+public class HideTapToPlaceLabel : MonoBehaviour
+{
+    // Start is called before the first frame update
+
+    private TapToPlace tapToPlace;
+    void Start()
+    {
+        tapToPlace = gameObject.GetComponent<TapToPlace>();
+        
+    }
+
+    // Update is called once per frame
```

Comment:
Nit: Get rid of the default Unity comments.

Reply:
removed